src/Sdk/Common/Common/VssHttpRequestSettings.cs

```diff
@@ -291,12 +291,12 @@ protected internal virtual Boolean IsHostLocal(String hostName)
         protected internal virtual Boolean ApplyTo(HttpRequestMessage request)
         {
             // Make sure we only apply the settings to the request once
-            if (request.Properties.ContainsKey(PropertyName))
+            if (request.Options.TryGetValue(new HttpRequestOptionsKey<VssHttpRequestSettings>(PropertyName), out _))
```

Comment:
There is some inconsistency in using `TryGetValue` method. In this place you use method defined on the `Options` class which takes `HttpRequestOptionsKey`. While in other places you use extension method from our code which accepts `string`. Most likely some namespaces are not imported in this file so extension method is not visible causing errors. You might consider using one or another approach throughout the code.

Reply:
fixed.